dev/integration_tests/ios_host_app/FlutterUITests/FlutterUITests.m

```diff
@@ -36,7 +36,12 @@ - (void)testFullScreenWarm {
     XCUIApplication *app = self.app;
 
     [self waitForAndTapElement:app.buttons[@"Full Screen (Warm)"]];
-    XCTAssertTrue([app.staticTexts[@"Button tapped 0 times."] waitForExistenceWithTimeout:60.0]);
+    BOOL newPageAppeared = [app.staticTexts[@"Button tapped 0 times."] waitForExistenceWithTimeout:60.0];
```

Comment:
I suspect 60 seconds is too long for everything in this test, but we can reevaluate once this test is stable.

Reply:
I refactored a little so all the 60.0 are now a constant, and yeah we can adjust it later.
Note that there are 2 time-outs in this tests have 30.0 time outs